extras/installer.sh

```diff
@@ -277,6 +277,9 @@ fi
 
 echo
 read -e -p "Directory to install into: " -i "/opt/plexupdate" FULL_PATH
+if [[ "$FULL_PATH" == *"~"* ]]; then # If path contains a tilde
```

Comment:
Instead, I tried

`FULL_PATH=$(eval echo ${FULL_PATH//[;&|]*/}`

which works quite well. The general gist is to simply use eval to properly deal with the tilde, but to also strip out anything following either a semi-colon, ampersand, or bar (which could be used to continue the statement). This eliminates people from entering things like

`~/test; cat /etc/passwd`
or 
`~user/test && cat /etc/passwd`